internal/backends/mcu/renderer.rs

```diff
@@ -812,39 +811,64 @@ impl i_slint_core::item_rendering::ItemRenderer for PrepareScene {
     }
 
     fn draw_text(&mut self, text: Pin<&i_slint_core::items::Text>) {
+        let max_width: PhysicalLength =
+            (LogicalLength::new(text.width()) * self.scale_factor).cast();
+        let max_height: PhysicalLength =
+            (LogicalLength::new(text.height()) * self.scale_factor).cast();
+
+        if max_width <= PhysicalLength::zero() || max_height <= PhysicalLength::zero() {
+            return;
+        }
+
         let font_request = text.unresolved_font_request().merge(&self.default_font);
-        let (font, glyphs) = crate::fonts::match_font(&font_request, self.scale_factor);
+        let font = crate::fonts::match_font(&font_request, self.scale_factor);
 
         let color = text.color().color();
 
-        let baseline_y = glyphs.ascent(font);
+        i_slint_core::textlayout::layout_text_lines(
+            &text.text(),
+            &font,
+            font.height(),
+            max_width,
+            max_height,
+            (text.horizontal_alignment(), text.vertical_alignment()),
+            text.wrap(),
+            text.overflow(),
+            false,
```

Comment:
I'm just reviewing that on github and it doesn't have "jump-to-definition" that makes it easy to go back.
So i'll just leave a comment saying that i don't know what that false is :-)

Reply:
It's `single_line: bool` :-)